test/lib/ufe/testComboCmd.py

```diff
@@ -890,6 +891,56 @@ def testCenterPivotUndo(self):
         assertVectorAlmostEqual(self, usdT3d.rotatePivot().vector, [0.0, 0.0, 0.0])
         assertVectorAlmostEqual(self, usdT3d.scalePivot().vector, [0.0, 0.0, 0.0])
 
+    @unittest.skipIf(int(cmds.about(apiVersion=True)) <= 20220000, 'Center pivot command is only available in Maya 2022 or later.')
+    def testCenterPivotUpdatePivotCompensations(self):
+        '''Center pivot must correctly update rotate, scale pivot compensations.'''
+
+        cmds.file(new=True, force=True)
+
+        import mayaUsd_createStageWithNewLayer
+        mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+        proxyShapeItem = ufeUtils.createItem('|stage1|stageShape1')
+        proxyShapeContextOps = ufe.ContextOps.contextOps(proxyShapeItem)
+        proxyShapeContextOps.doOp(['Add New Prim', 'Capsule'])
+
+        capsuleItem = ufeUtils.createItem('|stage1|stageShape1,/Capsule1')
+        t3d = ufe.Transform3d.transform3d(capsuleItem)
+
+        sn = ufe.GlobalSelection.get()
+        sn.clear()
+        sn.append(capsuleItem)
+
+        # Pivots and pivot compensations initially [0.0, 0.0, 0.0].
+        def checkNullPivotsAndCompensations():
+            for v in [t3d.rotatePivot(), t3d.scalePivot(), 
+                      t3d.rotatePivotTranslation(), 
+                      t3d.scalePivotTranslation()]:
+                assertVectorAlmostEqual(self, v.vector, [0.0, 0.0, 0.0])
+
+        checkNullPivotsAndCompensations()
+
+        # Rotate around x axis so that moving the pivot will create a rotate
+        # pivot translation as compensation, which keeps the total object
+        # transformation constant.
+        cmds.rotate(45, 0, 0, r=True, os=True, fo=True)
+
+        # Move the pivot.
+        cmds.move(0, 5, 0, r=True, urp=True, usp=True)
```

Comment:
I'm looking at the docs here: https://help.autodesk.com/cloudhelp/2023/CHS/Maya-Tech-Docs/CommandsPython/move.html and cannot see what urp and usp flags are. What are their long names? Pretty sure they must be pivot-related :)

Reply:
Yeah, unfortunately undocumented, which we should fix, ufeRotatePivot, ufeScalePivot.